previews/primer/alpha/select_panel_preview/custom_loading_label.html.erb

```diff
@@ -0,0 +1,18 @@
+<% subject_id = SecureRandom.hex %>
+
+<%= render(Primer::Alpha::SelectPanel.new(
+  data: { interaction_subject: subject_id },
+  src: select_panel_items_path(
+    select_variant: :multiple,
+    selected_items: selected_items,
+  ),
+  select_variant: :multiple,
+  fetch_strategy: :local,
```

Comment:
Could we add a comment here and in the other template explaining why we're providing a `src:` _and_ setting `fetch_strategy: :local`?